pycbc/workflow/grb_utils.py

```diff
@@ -515,20 +515,24 @@ def make_pygrb_plot(workflow, exec_name, out_dir,
     if exec_name == 'pygrb_efficiency':
```

Comment:
As a general comment, this function is a bit inelegant. Perhaps it would be neater to create a few short sub-classes of  `PlotExecutable` add the custom code for each executable there, and then `create_node` would call the relevant code. This is more in line with how the all-sky workflow handles this.